src/geode/io/model/msh_output.cpp

```diff
@@ -157,7 +157,25 @@ namespace
             geode::index_t count{ 1 };
             for( const auto& surface : brep_.surfaces() )
             {
-                write_component( surface, count++ );
```

Comment:
maybe write_component function is not used anymore -> delete it?

Reply:
It is used for Corner and Line